### DISCUSSION TITLE
Tesla API Outage

### DISCUSSION THREAD

**Discussion by zer0stars** [timdorr/tesla-api#784] [General]
Did Tesla shut off access to owner api today at 10AM EST?

**Comment by elffjs** [comment]
It looks like maybe only the streaming/websocket stuff is down.

**Comment by ravirajsinghrrs09** [comment]
Tesla power supply generally best . Sir I am from India to owner the Tesla owner supply manager Gmail contact ravirajsingh72846342@gmail.com 
. Thanks for meet sir .

**Comment by ravirajsinghrrs09** [comment]
Sir we have also visited Black Rocks company but we like Tesla very much so we want to accept it so please reply for my c***t @ravirajsinghrrs09 ravirajsingh72846342@gmail.com

- [ ]
